src/cartridges/int_adyen_overlay/cartridge/scripts/adyenGetOpenInvoiceData.js

```diff
@@ -45,7 +45,9 @@ function getLineItems(args) {
 
   // Add all product and shipping line items to request
   const lineItems = [];
-  for (const lineItem in order.getAllLineItems()) {
+  const allLineItems = order.getAllLineItems();
+  for (const i in allLineItems) {
```

Comment:
not sure if naming the variable i (counter) in a for..in statement is a good practice
What do you think @KadoBOT?

Reply:
Updated it to "item"